string user_name
---
float64 score_percentage
